mkdocs.yml

```diff
@@ -35,17 +35,19 @@ nav:
       - "Meta": "reference/meta.md"
       - "Storage": "reference/storage.md"
       - "Testing": "reference/testing.md"
+      - "Services":
```

Comment:
Right now, `services` folder gets rendered as a heading at equal level as "Reference CLI" in the TOC (see image), but better to have it at the same level as other CLI modules. So for now, maybe best to include tokenizer docs as `reference/tokenizer.md` rather than `reference/services/tokenizer.md`?

We can certainly reorganize this later if need be, but for now I'd stick with current hierarchy if not urgent to change.

<img width="352" height="732" alt="image" src="https://github.com/user-attachments/assets/74fad13d-295f-4c35-9d28-5e0c09551483" />